phoenix-core/src/main/java/org/apache/phoenix/coprocessor/tasks/DropChildViewsTask.java

```diff
@@ -46,6 +46,7 @@
     public TaskRegionObserver.TaskResult run(Task.TaskRecord taskRecord) {
         PhoenixConnection pconn = null;
         Timestamp timestamp = taskRecord.getTimeStamp();
+        LOGGER.info("DropChildViewTask is running for task " + taskRecord.getTableName());
```

Comment:
This was suggested for only debugging. Let's remove it or use LOGGER.debug() instead

Reply:
And the same comment for the other instances of LOGGER.info